docusaurus/docs/reference/tunnelers/linux/index.md

```diff
@@ -185,61 +181,87 @@ repo_gpgcheck=1
-
-  ```text
-  sudo chown -cR :ziti        /opt/openziti/etc/identities
-  sudo chmod -cR ug=rwX,o-rwx /opt/openziti/etc/identities
-  ```
-
 2. Enable and start the service
 
     ```text
     sudo systemctl enable --now ziti-edge-tunnel.service
     ```
 
-2. The process needs to be restarted if the contents of `/opt/openziti/etc/identities` change.
+2. Add an Identity
 
-    ```text
-    sudo systemctl restart ziti-edge-tunnel.service
-    ```
+## Adding an Identity
+
+### Restart to Load All in Identity Directory
+
+The tunneler will load all identities in the `--identity-dir` directory. The default location for identities is is
+`/opt/openziti/etc/identities`. Add identity files to this directory by copying the identity file into the directory and
+setting permissions for group `ziti` so the tunneler service may delete JWT enrollment tokens and read and write the
+JSON identity configuration files.
+
+```text
+sudo chown -cR :ziti        /opt/openziti/etc/identities
+sudo chmod -cR ug=rwX,o-rwx /opt/openziti/etc/identities
+```
+
+The tunneler process needs to be restarted if the contents of `/opt/openziti/etc/identities` change.
+
+```text
+sudo systemctl restart ziti-edge-tunnel.service
+```
+
+### Load a Single Identity Without Restarting
+
+This sends an IPC command to the running tunneler and requires write permission on the IPC socket
+(`/tmp/.ziti/ziti-edge-tunnel.sock`). Run this command as the logged in user instead of user `ziti` if a member of group
+`ziti`.
+
+```text
+sudo -u ziti \
+  ziti-edge-tunnel add --jwt "$(< ./in-file.jwt)" --identity myIdentityName
+```
 
 ## Manual Installation
 
-[The latest binary release](https://github.com/openziti/ziti-tunnel-sdk-c/releases/latest/) of `ziti-edge-tunnel` is distributed as an executable for amd64, arm, arm64 architectures. The upgrade procedure is identical to the installation procedure.
+[The latest binary release](https://github.com/openziti/ziti-tunnel-sdk-c/releases/latest/) of `ziti-edge-tunnel` is
+distributed as an executable for amd64, arm, arm64 architectures. The upgrade procedure is identical to the installation
+procedure.
 
-You'll need to install the `wget` and `unzip` commands to use this example.
+Install the `wget` and `unzip` commands to use this example.
 
 ```text
-wget -q "https://github.com/openziti/ziti-tunnel-sdk-c/releases/latest/download/ziti-edge-tunnel-Linux_$(uname -p).zip" \
+wget -q \
+  "https://github.com/openziti/ziti-tunnel-sdk-c/releases/latest/download/ziti-edge-tunnel-Linux_$(uname -p).zip" \
   && unzip ./ziti-edge-tunnel-Linux_$(uname -p).zip \
   && rm ./ziti-edge-tunnel-Linux_$(uname -p).zip \
   && chmod -c +x ./ziti-edge-tunnel \
   && ./ziti-edge-tunnel version
 ```
 
-### Enroll Before You Run
+### Enroll Before Run
 
-You will need the token file or its contents to enroll. Enrollment is the act of exchanging the token for an identity that is to be permanently installed in the filesystem.
+A one-time token (JWT) is required to enroll.
```

Comment:
all these sections should be restated, particularly if you agree with making 'manual' a separate install tab. that or enroll before run should just be a reference to 'identities' whichwould be better imo